contrib/babelfishpg_common/src/babelfishpg_common.c

```diff
@@ -149,6 +165,14 @@ _PG_init(void)
  	prev_get_like_collation_hook = get_like_collation_hook;
 	get_like_collation_hook = bbf_get_like_collation;
 
+	/* Initialize logical db name */
+	g_logical_db_cache.logical_db_name = NULL;
+	g_logical_db_cache.is_valid = false;
```

Comment:
I think it will be better to modify assign_guc_hook for psql_logical_babelfish_db_name so that we can distinguish NULL and not set GUC.